### PR TITLE
Gallery: Use Horse to illustrate Convex Hull

### PR DESCRIPTION
**Pull request by sciunto**
## Description

These modifications are on my disk for a while. I had to push the invert() function and a fix on the horse picture and I forgot in the mean time this patch.

Here, I propose to use the horse to illustrate the convex hull transformation. To me, this illustration is much more straightforward than the previous one. In the original version, getting a int picture with additional gray pixels from a binary image as an input is visually puzzling  to me. 
This new version also has the advantage to show the use of invert().

The idea is to improve again our examples for the next release.

sphx_glr_plot_convex_hull_001

### CODE REVIEW

File: doc/examples/edges/plot_convex_hull.py [hunk 1]
```diff
@@ -18,38 +15,23 @@
...
-image = np.array(
-    [[0, 0, 0, 0, 0, 0, 0, 0, 0],
-     [0, 0, 0, 0, 1, 0, 0, 0, 0],
-     [0, 0, 0, 1, 0, 1, 0, 0, 0],
-     [0, 0, 1, 0, 0, 0, 1, 0, 0],
-     [0, 1, 0, 0, 0, 0, 0, 1, 0],
-     [0, 0, 0, 0, 0, 0, 0, 0, 0]], dtype=float)
-
-original_image = np.copy(image)
+# The original image is inverted as the object must be white.
```
Comment (soupault) [inline doc/examples/edges/plot_convex_hull.py]: -> "Invert the image so to make horse silhouette have higher intensity values"?

File: doc/examples/edges/plot_convex_hull.py [hunk 2]
```diff
@@ -18,38 +15,23 @@
...
+image = invert(data.horse())
 
 chull = convex_hull_image(image)
-image[chull] += 1
```
Comment (soupault) [inline doc/examples/edges/plot_convex_hull.py]: I really like the way the current example looks.
@sciunto Have you tried to add an 1-pixel contour of the horse to the second plot? For the horse image I think it should be less 'puzzling'. :thinking:

File: doc/examples/edges/plot_convex_hull.py (hunk 1 above)
Comment (sciunto) [inline doc/examples/edges/plot_convex_hull.py]: But Convex-hull works only on binary images. Do you think that "higher" is appropriated?

File: doc/examples/edges/plot_convex_hull.py (hunk 2 above)
Comment (sciunto) [inline doc/examples/edges/plot_convex_hull.py]: OK. I think I'll try to make a second plot to better explain that gray = added pixels.

Comment (sciunto) [inline doc/examples/edges/plot_convex_hull.py]: done

File: doc/examples/edges/plot_convex_hull.py (hunk 1 above)
Comment (soupault) [inline doc/examples/edges/plot_convex_hull.py]: I just think that it is worthwhile to mention the correspondence between intensities and colors here.
From the first read this sentence was very unclear to me: object be white -> it was black? -> what does it mean to be white? -> what does invert actually do?. I think if you could clarify this a bit, it wil be perfect!

**Comment by sciunto** [comment]
@emmanuelle Do you have time to make the second review?

**Comment by jni** [comment]
Thanks @sciunto and thanks @soupault and @alexandrejaguar for the reviews!

**Pull request merged**